CBMT/IntroCalificacionesViewController.swift

```diff
@@ -24,6 +24,8 @@ class IntroCalificacionesViewController: UIViewController, UITableViewDataSource
         Alumno(nombre: "Lucia Cantu")
     ]
```

Comment:
btw- this array of Alumnos is a temporary solution until we get backend resolved